promotion/suggestions.go

```diff
@@ -166,8 +171,58 @@ func (service *Service) RedeemAndCreateSuggestionEvent(ctx context.Context, cred
 		return nil
 	}
 
-	service.eventChannel <- suggestion
-	// TODO emit event
+	// kafka event producer below - this could probably put into a generic module
+	kafkaBrokers := os.Getenv("KAFKA_BROKERS_STRING")
+	kafkaWriter := kafka.NewWriter(kafka.WriterConfig{
+		// by default we are waitng for acks from all nodes
+		Brokers:  []string{kafkaBrokers},
+		Topic:    "suggestion",
+		Balancer: &kafka.LeastBytes{},
+	})
+	defer closers.Panic(kafkaWriter)
+
+	// possible that this schema could be tightened more
+	// also, are all fields required?
+	// we re-read this file every time which is decidedly not fantastic
```

Comment:
agreed - docs say that the `Codec` is stateless and safe for use from multiple goroutines. so we could load this once into some struct embedded in the service?